ParcelInvoiceMaker/ParcelInvoiceMaker/Provider/ReceiptProvider.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  ReceiptProvider.swift
+//  ParcelInvoiceMaker
+//
+//  Created by MacBook on 2024/03/11.
+//
+
+import Foundation
+
+protocol ReceiptProvider {
+    func canSend(_ method: ReceiptProvideMethod) -> Bool
+    func send(content: String) async
+}
+
+struct EmailReceiptProvider: ReceiptProvider {
+    func canSend(_ method: ReceiptProvideMethod) -> Bool {
+        method == .email
+    }
+    
+    func send(content: String) async {
+        Task {
+            print("이메일 영수증 전송: \(content)")
+        }
+    }
```

Comment:
```suggestion
   func send(content: String) async {
        print("이메일 영수증 전송: \(content)")
    }
```
process쪽이 async await을 사용하게 된다면 해당 부분에는 Task를 사용해도 되지 않을것 같아요.